components/scrapd-home-section/component.js

```diff
@@ -31,13 +36,17 @@ const ScrapdHomeSection = props => {
     padding: '1em'
   }));
 
-  const Text = styled.div({
-    maxWidth: '75%',
-    minWidth: '75%'
-  });
+  const Text = styled('div')`
+    ${mq({
+      maxWidth: ['99%', '80%'],
+      minWidth: ['99%', '80%'],
+      fontSize: ['1em', '1.6em'],
```

Comment:
`1.6em` for the desktop mode seems a bit too big. You should play with other values, but I think `1.25em` should be enough.

Reply:
Totally agree, yikes.

Reply:
Completed.